ftl/common/constants.py

```diff
@@ -27,3 +27,16 @@
 PACKAGE_JSON = 'package.json'
 NODE_DEFAULT_ENTRYPOINT = 'node server.js'
 NPMRC = '.npmrc'
+
+P1_CACHE_STR = '{key_version}:{language}->{key}'
+P2_CACHE_STR = '{key_version}:{language}:{package_name}:' \
+            '{package_version}->{key}'
+CACHE_HIT = '[CACHE][HIT] '
+CACHE_MISS = '[CACHE][MISS] '
+
+P1_CACHE_HIT_LOGSTR = CACHE_HIT + P1_CACHE_STR
+P2_CACHE_HIT_LOGSTR = CACHE_HIT + P2_CACHE_STR
+P1_CACHE_MISS_LOGSTR = CACHE_MISS + P1_CACHE_STR
+P2_CACHE_MISS_LOGSTR = CACHE_MISS + P2_CACHE_STR
+
+CACHE_KEY_VERSION = 'v1.0'
```

Comment:
we have a similar value _<RUNTIME>_CACHE_VERSION passed into the builder/cache.  might make sense to consolidate them